list.js

```diff
@@ -0,0 +1,16 @@
+// document.querySelector("#eraser").addEventListener(“click” () => {} )
+
+const input = document.getElementsByClassName("input");
+
+const addPoint = () => {
+    const point = document.createElement("h2");
+    point.textContent = input.textContent;
+
+    point.addEventListener("click", getLineThrough);
+
+    const getLineThrough = () => (point.style.textDecoration != "line-trough") ? point.style.textDecoration = "line-trough" : point.style.textDecoration = "none";
```

Comment:
1 strict comparison
2 
```suggestion
    const getLineThrough = () => {
    const lineThroughStyle = "line-trough";
    const noneStyle = "none";
        
    const isLineThrough = point.style.textDecoration === lineThroughStyle;
    
 point.style.textDecoration = isLineThrough ? noneStyle : lineThroughStyle;
```